server/fleet/hosts.go

```diff
@@ -538,9 +538,9 @@ func (h *Host) IsDEPAssignedToFleet() bool {
 	return h.DEPAssignedToFleet != nil && *h.DEPAssignedToFleet
 }
 
-// IsElegibleForDEPMigration returns true if the host fulfills all requirements
+// IsEligibleForDEPMigration returns true if the host fulfills all requirements
```

Comment:
Not a native speaker but according to merriam the proper syntax is `eligible`, not `elegible` :)

Reply:
I think this is my bad, thanks for fixing it! `elegible` sounds good in spanish 😅

Reply:
@roperzh It sounded good to me too! I added a couple of those before noticing this :D